dockers/docker-database/Dockerfile.j2

```diff
@@ -11,21 +11,22 @@ ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get update
 
 # Install redis-server
+# TODO: Require direct bullseye-security installation after the OS release version upgraded
 {% if CONFIGURED_ARCH == "armhf" %}
-RUN curl -k -o redis-tools_6.0.6-1~bpo10+1_armhf.deb "https://sonicstorage.blob.core.windows.net/packages/redis/redis-tools_6.0.6-1_bpo10+1_armhf.deb?sv=2015-04-05&sr=b&sig=67vHAMxsl%2BS3X1KsqhdYhakJkGdg5FKSPgU8kUiw4as%3D&se=2030-10-24T04%3A22%3A40Z&sp=r"
-RUN curl -k -o redis-server_6.0.6-1~bpo10+1_armhf.deb "https://sonicstorage.blob.core.windows.net/packages/redis/redis-server_6.0.6-1_bpo10+1_armhf.deb?sv=2015-04-05&sr=b&sig=xTdayvm0RBguxi9suyv855jKRjU%2FmKQ8nHuct4WSX%2FA%3D&se=2030-10-24T04%3A22%3A05Z&sp=r"
-RUN dpkg -i redis-tools_6.0.6-1~bpo10+1_armhf.deb redis-server_6.0.6-1~bpo10+1_armhf.deb || apt-get install -f
-RUN rm redis-tools_6.0.6-1~bpo10+1_armhf.deb redis-server_6.0.6-1~bpo10+1_armhf.deb
+RUN curl -sS -o redis-tools_6.0.15-1_armhf.deb "http://ftp.debian.org/debian/pool/main/r/redis/redis-tools_6.0.15-1_armhf.deb"    && \
```

Comment:
I think this can be changed to use apt now, since we're getting from the Debian repos?

Reply:
Yes, apt-get can handle all the jobs now.